src/components/connection-prompt/ConnectDialogContainer.svelte

```diff
@@ -124,11 +128,21 @@
         }
         flashProgress = progress;
       });
-      // Finished flashing successfully
+
+      // Store radio/bluetooth details. Radio is essential to pass to micro:bit 2.
+      // Bluetooth saves the user from entering the pattern.
+      if (flashStage === 'bluetooth') {
+        $btPatternInput = MBSpecs.Utility.nameToPattern(
+          MBSpecs.Utility.serialNumberToName(deviceId),
+        );
+      }
+      if (flashStage === 'radio-sender') {
+        // TODO: check max value, add constant etc.
+        $radioBridgeFrequency = 42; // deviceId % 83;
```

Comment:
@microbit-carlos if you update the hex and this code to calculate the frequency in the same way then we think it will work (and be persisted).

Reply:
I need to add another command to the micro:bit 2 bridge to configure the micro:bit unique ID to listen to (so the micro:bit 1 remote unique ID).
This is because we are using the combination of frequency + micro:bit ID, which is sent by the remote on each radio packet.

Reply:
Sorry, this comment is more about the `await Microbits.assignSerialInput(usb, $radioBridgeFrequency);` line than this one, but mostly to share that the micro:bit ID is something we need to track and configure in MicrobitSerial as well.

Reply:
Switched to just pass the device id in 3a9f68d235309fd556b6b16c3b52619c8ca5962b. We can derive the frequency from that either on device (to be consistent with the next gen plan for the remote micro:bit) or in MicrobitSerial.